config/owasp/suppressions.xml

```diff
@@ -603,140 +521,23 @@
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-core@.*$</packageUrl>
-        <cpe>cpe:/a:pivotal_software:spring_security</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: dom4j-2.1.1.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.dom4j/dom4j@.*$</packageUrl>
-        <cpe>cpe:/a:dom4j_project:dom4j</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: javax.websocket-api-1.1.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/javax\.websocket/javax\.websocket\-api@.*$</packageUrl>
-        <cve>CVE-2020-11050</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: spring-security-oauth2-core-5.2.1.RELEASE.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-core@.*$</packageUrl>
-        <cpe>cpe:/a:pivotal_software:spring_security_oauth</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: spring-security-oauth2-client-5.2.1.RELEASE.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-client@.*$</packageUrl>
-        <cve>CVE-2018-1258</cve>
-        <cve>CVE-2020-5407</cve>
-        <cve>CVE-2020-5408</cve>
+        <cpe>cpe:/a:gitlab:gitlab</cpe>
+        <cve>CVE-2013-4580</cve>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: spring-cloud-contract-wiremock-2.2.1.RELEASE.jar
+   file name: log4j-to-slf4j-2.12.1.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework\.cloud/spring\-cloud\-contract\-wiremock@.*$</packageUrl>
-        <cve>CVE-2018-9116</cve>
-        <cve>CVE-2018-9117</cve>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.logging\.log4j/log4j\-to\-slf4j@.*$</packageUrl>
+        <cve>CVE-2020-9488</cve>
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: wiremock-jre8-standalone-2.25.1.jar
+   file name: log4j-api-2.12.1.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
-        <cve>CVE-2019-16942</cve>
-        <cve>CVE-2019-16943</cve>
-        <cve>CVE-2019-17531</cve>
-        <cve>CVE-2019-20330</cve>
-        <cve>CVE-2020-10672</cve>
-        <cve>CVE-2020-10673</cve>
-        <cve>CVE-2020-10968</cve>
-        <cve>CVE-2020-10969</cve>
-        <cve>CVE-2020-11111</cve>
-        <cve>CVE-2020-11112</cve>
-        <cve>CVE-2020-11113</cve>
-        <cve>CVE-2020-11619</cve>
-        <cve>CVE-2020-11620</cve>
-        <cve>CVE-2020-8840</cve>
-        <cve>CVE-2020-9546</cve>
-        <cve>CVE-2020-9548</cve>
-        <cve>CVE-2020-9547</cve>
+        <packageUrl regex="true">^pkg:maven/org\.apache\.logging\.log4j/log4j\-api@.*$</packageUrl>
+        <cve>CVE-2020-9488</cve>
```

Comment:
There seems to be quite a large amount of vulnerabilities in this PR which will go into Demo if not suppressed: https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/job/PR-596/8/artifact/build/reports/dependency-check-report.html

Reply:
Are they not already suppressed in master? if yes then why are they missing in PR?

Reply:
PR file and master file are identical. So all master changes are in PR. I think that's ok.

Reply:
Perhaps these are new vulnerabilities then - either way we should we not suppress them?

Reply:
This has been resolved in the following commit: https://github.com/hmcts/rd-professional-api/pull/596/commits/cf92314f27b1332f0c4dec152e800b4d0be761c3